src/rgw/services/svc_bilog_rados.h

```diff
@@ -35,22 +35,22 @@ class RGWSI_BILog_RADOS : public RGWServiceInstance
 
   void init(RGWSI_BucketIndex_RADOS *bi_rados_svc);
 
-  int log_start(const RGWBucketInfo& bucket_info, int shard_id);
-  int log_stop(const RGWBucketInfo& bucket_info, int shard_id);
+  int log_start(const RGWBucketInfo& bucket_info, int shard_id, const bucket_log_layout_generation& log_layout);
+  int log_stop(const RGWBucketInfo& bucket_info, int shard_id, const bucket_log_layout_generation& log_layout);
```

Comment:
these bucket layout types are in `namespace rgw` so the argument needs to be qualified as `rgw::bucket_log_layout_generation`. and please put the layout argument before the `int shard_id` argument in these functions